docs/en/latest/tutorials/add-multiple-api-versions.md

```diff
@@ -0,0 +1,227 @@
+
+## Types of API Versioning
+
+#### URI Path
+
+The most common way to version an API is in the URI path and is often done with the prefix "v". This method employs URI routing to direct requests to a specific version of the API.
+
+```shell
+http://org.apisix/v1/hello
+http://org.apisix/v2/hello
+```
+
+#### Query Params
+
+In this method, the version number is included in the URI, but as a query parameter instead of in the path.
+
+```shell
+http://org.apisix/hello?version=1
+http://org.apisix/hello?version=2
+```
+
+#### Custom Request Header
+
+You can also set the version number using custom headers in requests and responses. This leaves the URI of your resources unchanged.
+
+```shell
+http://org.apisix/hello -H 'Version: 1'
+http://org.apisix/hello -H 'Version: 2'
+```
+
+The primary goal of versioning is to provide users of an API with the most functionality possible while causing minimal inconvenience. Keeping this goal in mind, let’s have a look in this tutorial at how to _publish and manage multiple versions of your API_ with Apache APISIX.
+
+**In this tutorial**, you learn how to:
+
+- Create a route and upstream for our sample API.
+- Add a new version to the existing API.
+- Use [proxy-rewrite](https://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/) plugin to rewrite the path in a plugin configuration.
+- Route API requests from the old version to the new one.
+
+## Prerequisites
+
+For the demo case, we will leverage the sample repository [Evolve APIs](https://github.com/nfrankel/evolve-apis) on GitHub built on the Spring boot that demonstrates our API. You can see the complete source code there.
+
+To execute and customize the example project per your need shown in this tutorial, here are the minimum requirements you need to install in your system:
+
+- [Docker Desktop](https://docs.docker.com/desktop/windows/install/) - you need [Docker desktop](https://www.docker.com/products/docker-desktop/) installed locally to complete this tutorial. It is available for [Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe) or [macOS](https://desktop.docker.com/mac/edge/Docker.dmg).
+
+Also, complete the following steps to run the sample project with Docker.
+
+Use [git](https://git-scm.com/downloads) to clone the repository:
+
+``` shell
+git clone 'https://github.com/nfrankel/evolve-apis'
+```
+
+Go to root directory of _evolve-apis_
+
+``` shell
+cd evolve-apis
+```
+
+Now we can start our application by running `docker compose up` command from the root folder of the project:
+
+``` shell
+docker compose up -d
+```
+
+### Create a route and upstream for the API.
+
+You first need to [route](https://apisix.apache.org/docs/apisix/terminology/route/) your HTTP requests from the gateway to an [upstream](https://apisix.apache.org/docs/apisix/terminology/upstream/) (your API). With APISIX, you can create a route by sending an HTTP request to the gateway.
```

Comment:
In all of our docs, we mention APISIX specific terminology with uppercase. ~route~ Route, ~upstream~ Upstream. See: https://apisix.apache.org/docs/general/documentation-style-guide/#language

Reply:
Sure, fixed it.